adanet/core/estimator_distributed_test_runner.py

```diff
@@ -288,6 +289,35 @@ def tree_loss_fn(labels, logits):
     estimator = AutoEnsembleEstimator(
         head=head, candidate_pool=candidate_pool, **kwargs)
 
+  elif FLAGS.estimator_type == "estimator_with_experimental_multiworker_strategy":
+    def _model_fn(features, labels, mode):
+      layer = tf.layers.Dense(1)
+      logits = layer(features)
+
+      if mode == tf.estimator.ModeKeys.PREDICT:
+        predictions = {"logits": logits}
+        return tf.estimator.EstimatorSpec(mode, predictions=predictions)
+
+      loss = tf.losses.mean_squared_error(labels=labels, predictions=tf.reshape(logits,[]))
+
+      if mode == tf.estimator.ModeKeys.EVAL:
+        tf.estimator.EstimatorSpec(mode, loss=loss)
+
+      if mode == tf.estimator.ModeKeys.TRAIN:
+        train_op = tf.train.GradientDescentOptimizer(0.2).minimize(loss)
+        return tf.EstimatorSpec(mode, loss=loss, train_op=train_op)
+
+    def _input_fn():
+      features = tf.data.Dataset.from_tensors([[1.]]).repeat(100)
+      labels = tf.data.Dataset.from_tensors(1.).repeat(100)
+      return tf.data.Dataset.zip((features, labels))
+
+    distribution = tf.distribute.experimental.MultiWorkerMirroredStrategy()
+    config = tf.estimator.RunConfig(train_distribute=distribution)
+    classifier = tf.estimator.Estimator(model_fn=_model_fn, config=config)
+    classifier.train(input_fn=_input_fn)
```

Comment:
Delete these two lines, since the training will happen below in `tf.estimator.train_and_evaluate(estimator, train_spec, eval_spec)`